content/cucumber/api.md

```diff
@@ -772,6 +772,53 @@ import org.junit.runner.RunWith;
 public class RunCukesTest {
 }
 ```
+For example if you want to check whether all feature file steps has corresponding step definitions, you can specify it like this: 
+
+```java
+package mypackage;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(dryRun=true)
+public class RunCukesTest {
+}
+```
+dryRun default value is false.
```

Comment:
Please use a whole sentence starting with a capital letter, optionally marking the option and it's value, for instance: "The default option for `dryRun` is `false`. (Note: please do this consistently for all three options)